### PR TITLE
Added caption

### PR DESCRIPTION
**Pull request by chrisn555-im**

### CODE REVIEW

**Comment by xxcutive** [comment]
why?

**Review by xxcutive — APPROVED** [review]
Instructor cannot be wrong

**Comment by d-d-fisher** [comment]
comment

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # GitHub for Developers
 
 - Class Date: June 27-28, 2024
-- Facilitator: @chrisn555-im and others
+- Facilitator: @chrisn555-im
```
Comment (liqin06) [inline README.md:5]: test

**Review by Sgregory2024 — APPROVED** [review]

**Pull request merged**